src/dune_rules/install_rules.ml

```diff
@@ -42,6 +48,17 @@ module Package_paths = struct
     Path.Build.relative (build_dir ctx pkg) (Package.Name.to_string name ^ ".dune-package")
   ;;
 
+  let odoc_config_file ctx pkg =
+    if need_odoc_config pkg
+    then (
+      let name = Package.name pkg in
+      Some
+        (Path.Build.relative
+           (build_dir ctx pkg)
+           (Package.Name.to_string name ^ ".odoc-config.sexp")))
```

Comment:
I feel `sprintf "%s.odoc-config.sexp" name` might be a bit nicer here, but it's very much nitpicking.